p4proto/p4src/linux_networking/README_LINUX_NETWORKING.md

```diff
@@ -208,7 +227,48 @@ Note: Port number used in ovs-p4ctl commands are target datapath indexes(unique
     ovs-p4ctl add-entry br0 linux_networking_control.handle_tx_control_pkts_table "istd.input_port=5,action=linux_networking_control.set_control_dest(4)"
   ```
 
-#### 13. Test the ping scenarios:
+#### 13. Configure routes only when TEP port used for tunnel termination.
```

Comment:
Route information is populated either statically or dynamically using FRR BGP.
Make sure the section mentioned for route configuration conveys the same and not use the procedures mentioned for both approaches. Same comment applies for ECMP configuration README too.